### PR TITLE
Improve Tokens::clearEmptyTokens performance

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

**Comment by keradus** [comment]
merged

File: Symfony/CS/Tokenizer/Tokens.php [hunk 1]
```diff
@@ -304,9 +304,23 @@ public function clearChanged()
      */
     public function clearEmptyTokens()
     {
-        $count = 0;
+        $limit = $this->count();
+        $index = 0;
+
+        for ($index = 0; $index < $limit; ++$index) {
```
Comment (stof) [inline Symfony/CS/Tokenizer/Tokens.php]: ``$index=0`` is not necessary here as you initialize it above (ot the above initialization is useless)

Comment (keradus) [inline Symfony/CS/Tokenizer/Tokens.php]: good point

Comment (keradus) [inline Symfony/CS/Tokenizer/Tokens.php]: #981

**Pull request merged**